packages/dd-trace/test/exporters/common/docker.spec.js

```diff
@@ -35,7 +35,7 @@ describe('docker', () => {
       `1:name=systemd:/docker/${id}`
     ].join('\n')
 
-    fs.readFileSync.withArgs('/proc/self/cgroup').returns(Buffer.from(cgroup))
+    fs.readFileSync.withArgs('/proc/self/cgroup').returns(cgroup)
```

Comment:
`readFileSync` only returns a string with the encoding specified, so let's add the encoding to the test too.

Reply:
Good catch! Thank you for pointing that out!